ui/admin/app/templates/application.hbs

```diff
@@ -5,113 +5,23 @@
 
 {{page-title (app-name)}}
 
-<Rose::Layout::Global as |layout|>
+<Hds::AppFrame @hasHeader={{false}} as |Frame|>
```

Comment:
Not sure how it works, but if you bring the same changes to the "desktop" app and remove/replace the `<Rose::Layout::Global>` instance from [the playground.hbs file](https://github.com/hashicorp/boundary-ui/blob/main/addons/rose/tests/dummy/app/templates/playground.hbs), at that point you can remove entirely the code (hbs, css, js, etc) of the `Rose::Layout::Global` component 

_(not sure if that would be outside of the scope of this PR)_

Reply:
Similarly for the `Rose::Layout::Sidebar` which is only left in the playground and in [this page](https://github.com/hashicorp/boundary-ui/blob/main/ui/desktop/app/templates/scopes/scope/projects.hbs)

Reply:
(the reason I know it, is because in those days I am doing an audit of all the page/layout components used in our products codebases, and I noticed in Boundary these Two rose components (`Rose::Layout::Global` and `Rose::Layout::Sidebar`) and under each one of them, in my audit document, I've written: _"Likely the combination of Global+Sidebar layout components can be replaced with a single Hds::AppFrame component"_ 🙂